stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -41,7 +41,7 @@ public class _AnyTensorHandle {
 /// into a tensor program.
 @_fixed_layout // required because the compiler accesses _cTensorHandle directly.
 public final class TensorHandle<Scalar> : _AnyTensorHandle
-  where Scalar : AccelerableByTensorFlow {
+  where Scalar : _TensorFlowDataTypeCompatible {
```

Comment:
Yes, `TensorHandle` supports string elements. Good point about not conflating strings with scalars. But given that `TensorHandle` is not a user-facing type, we can add a TODO here and address it later.

Reply:
Yeah, Strings are allowed here. What if we rename `TensorHandle`'s generic parameter to `Element`? And also rename `_tf_string_tensor_from_scalars` to `_tf_string_tensor_from_elements`?

Reply:
Or actually, `_tf_string_tensor_from_scalars` could be `_tf_string_tensor_from_strings` regardless of whether we rename `TensorHandle`'s parameter to `Element`. I'll rename the `__tf_string_tensor_from_*` fns now and wait for more opinions before renaming  `TensorHandle`'s parameter to `Element`.

Reply:
I'd rather rename `TensorHandle` to `_TensorHandle` because it's never a user-facing type. When that is done, calling it "Scalar" is fine because it *is* a scalar in the backend (TF).